warp/tests/cuda/test_graph_set_conditional.py

```
@@ -0,0 +1,140 @@
+but WITHOUT Warp's trailing condition-evaluation kernel: the user kernel's
+wp.graph_set_conditional() call is the only thing driving loop-back, exactly
+as in an externally owned scope.
+"""
+
+import ctypes
+import unittest
+
+import warp as wp
+import warp._src.context as wp_context
+from warp._src.context import capture_pause, capture_resume
+from warp.tests.unittest_utils import *
```

Ruff (0.16.1)

[error] `from warp.tests.unittest_utils import *` used; unable to detect undefined names

(F403)